### ISSUE TITLE
Can't handle kotlin's collections

### ISSUE THREAD

**Issue by Nylle** [Nylle/KotlinFixture#1]
It's not a surprise, that KotlinFixture cannot handle `kotlin.collections`, since it's only a wrapper for JavaFixture.

The two following example will return a `java.util.ArrayList` where all elements are `Object`:
```
val result = Fixture().create<List<Int>>()
val result = Fixture().create<List<Integer>>()
```
This will work:
```
val result = Fixture().create<java.util.ArrayList<Int>>()
```

**Comment by Nylle** [comment]
This is actually not a missing feature, but a [bug](https://github.com/Nylle/JavaFixture/issues/57) in the underlying JavaFixture.

**Comment by Nylle** [comment]
This will be fixed in release 0.2.0.
